src/SFA.DAS.ApplyService.Application/Organisations/CreateOrganisation/CreateOrganisationHandler.cs

```diff
@@ -25,12 +27,16 @@ public async Task<Organisation> Handle(CreateOrganisationRequest request, Cancel
 
             var result = await UpdateOrganisationIfExists(request) ?? await CreateNewOrganisation(request);
```

Comment:
Why are we updating in create when there is a separate update handler. This needs sorting out. Find the usage of both the handlers and see if you can either keep only one handler (upsert) or avoid updating in create handler so consumer determines whether to update or create and calls respective handler. I prefer the former approach of having a single handler.

Reply:
UpdateOrganisationHandler is deleted and CreateOrganisationHandler is renamed to ManageOrganisationHandler